### PR TITLE
Update workflow for module

### PR DESCRIPTION
**Pull request by huiwangredhat**
[OLCS-74](https://issues.redhat.com/browse/OLCS-74)
Fixes OLCS-74

### CODE REVIEW

File: openlcsd/flow/tasks.py [hunk 1]
```diff
@@ -944,27 +944,48 @@ def get_module_components_from_corgi(context, engine):
...
+
 def fork_detail_type_components_imports(
         context, engine, nvr_list, src_dir, comp_type):
     """
     In a source container, it has different type of component. The different
-    components have different src_dir. This function will be defined the post
-    data for the different type component.
+    components have different src dir. This function will be defined the post
+    data for the source container srpm components' tasks. It could also be
+    same with rhel module srpm components except the src dir.
```
Comment (hanchuntao) [inline openlcsd/flow/tasks.py:975]: I cannot understand what is `except the src dir`

File: openlcsd/flow/tasks.py [hunk 2]
```diff
@@ -944,27 +944,48 @@ def get_module_components_from_corgi(context, engine):
...
+    """
+    components = context.get('components')
+    rpm_components = components.get('RPM')
+    engine.logger.info("Start to get srpm builds of the module...")
+    srpms = []
+    for rpm in rpm_components:
+        if rpm.get('arch') == 'src':
+            nvr = [rpm.get('name'), '-', rpm.get('version'), '-',
+                   rpm.get('release')]
+            srpms.append(''.join(nvr))
```
Comment (hanchuntao) [inline openlcsd/flow/tasks.py:964]: It is better to change the code as follows:
```
            nvr = "-".join([rpm.get('name'), rpm.get('version'), rpm.get('release')])
            srpms.append(nvr)
```

**Comment by hanchuntao** [comment]
It is better to add the corgi PR where they change the type in the code

File: openlcsd/flow/tasks.py (hunk 1 above)
Comment (huiwangredhat) [reply openlcsd/flow/tasks.py:975]: The src dir is the source directory. If the components are the container child components, they have been downloaded when the source image had been downloaded. The src dir should be defined in container import. If the components are the module child components, they are the general srpms downloading that the src dir is different for the forked task.

File: openlcsd/flow/tasks.py (hunk 2 above)
Comment (huiwangredhat) [reply openlcsd/flow/tasks.py:964]: Sure, will change.

**Pull request closed without merging**